app/assets/scripts/actions/action-creators.js

```diff
@@ -437,3 +443,99 @@ export function setGlobalZoom (zoomSource) {
     json: json
   };
 }
+
+function requestVProMMsBbox () {
+  return {
+    type: actions.REQUEST_VPROMMS_BBOX
+  };
+}
+
+function recieveVProMMsBbox (json) {
+  return {
+    type: actions.RECIEVE_VPROMMS_BBOX,
+    json: json
+  };
+}
+export function fetchVProMMsBbox (vprommsId) {
+  return function (dispatch) {
+    dispatch(requestVProMMsBbox());
+    let url = `${config.api}/way/${vprommsId}/bbox`;
+    return fetch(url)
+    .then(response => {
+      return response.json();
+    })
+    .then(json => {
+      if (json.statusCode === 400) {
+        return {dataAvailable: false};
```

Comment:
This is not being returned to anything. Since it's not being dispatched as an error to the state how are you capturing this?